frame/society/src/lib.rs

```diff
@@ -68,6 +68,14 @@
 //! The membership society is independently funded by a treasury managed by this
 //! module. Some subset of this treasury is placed in a Society Pot, which is used
 //! to determine the number of accepted bids.
+//!
+//! #### Rate of Growth
+//! 
+//! The membership society can grow at a rate of 10 accepted candidates per rotation period up
+//! to the max membership threshold. Once this threshold is met, candidate selections
+//! are stalled until there is space for new members to join. This can be resolved by
+//! voting out existing members through the random challenges or by using governance
+//! to increase the maximum membership count.
```

Comment:
Note the behavior when max membership is reached.

It is possible that we have `max_membership + 10` total users in the member list. This is because the approved candidates have clean way to "fail" being added to the membership pool.

The scenario can happen as so:
* There are 90 active members in society
* There are at least 10 suspended members in society
* There are 10 approved candidates that will make it in during the next rotation period
* The suspension judgement origin adds 10 suspended members back to the society, reaching the limit of 100
* Then the rotation period occurs and 10 candidates are added on top

As mentioned with inline comments, this has no consensus critical issue or side effects. Being over the max or being at the max is exactly the same behavior.